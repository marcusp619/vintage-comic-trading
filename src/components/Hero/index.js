import React from 'react';
import styled from 'styled-components';

const Hero = () => {
  return <HeroSection />;
};

export default Hero;

const HeroSection = styled.section`
  background: url('https://images.unsplash.com/photo-1521714161819-15534968fc5f?ixlib=rb-0.3.5&ixid=eyJhcHBfaWQiOjEyMDd9&s=4ed09f54e45802124a212b6163dec174&auto=format&fit=crop&w=3300&q=80')
    no-repeat center center fixed;
  background-size: cover;
  height: 300px;
`;
