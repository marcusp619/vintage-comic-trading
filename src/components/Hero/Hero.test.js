import React from 'react';

import {shallow} from 'enzyme';

import Hero from './index';

describe('Hero', () => {
  it('Hero should match snapshot', () => {
    const wrapper = shallow(<Hero />);
    expect(wrapper).toMatchSnapshot();
  });
});
