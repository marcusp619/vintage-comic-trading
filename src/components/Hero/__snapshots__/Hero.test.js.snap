// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Hero Hero should match snapshot 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Hero />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "function",
    "props": Object {},
    "ref": null,
    "rendered": null,
    "type": Object {
      "$$typeof": Symbol(react.forward_ref),
      "attrs": undefined,
      "componentStyle": ComponentStyle {
        "componentId": "sc-bdVaJa",
        "isStatic": true,
        "rules": Array [
          "
  background: url('https://images.unsplash.com/photo-1521714161819-15534968fc5f?ixlib=rb-0.3.5&ixid=eyJhcHBfaWQiOjEyMDd9&s=4ed09f54e45802124a212b6163dec174&auto=format&fit=crop&w=3300&q=80')
    no-repeat center center fixed;
  background-size: cover;
  height: 300px;
",
        ],
      },
      "displayName": "styled.section",
      "render": [Function],
      "styledComponentId": "sc-bdVaJa",
      "target": "section",
      "toString": [Function],
      "warnTooManyClasses": [Function],
      "withComponent": [Function],
    },
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "function",
      "props": Object {},
      "ref": null,
      "rendered": null,
      "type": Object {
        "$$typeof": Symbol(react.forward_ref),
        "attrs": undefined,
        "componentStyle": ComponentStyle {
          "componentId": "sc-bdVaJa",
          "isStatic": true,
          "rules": Array [
            "
  background: url('https://images.unsplash.com/photo-1521714161819-15534968fc5f?ixlib=rb-0.3.5&ixid=eyJhcHBfaWQiOjEyMDd9&s=4ed09f54e45802124a212b6163dec174&auto=format&fit=crop&w=3300&q=80')
    no-repeat center center fixed;
  background-size: cover;
  height: 300px;
",
          ],
        },
        "displayName": "styled.section",
        "render": [Function],
        "styledComponentId": "sc-bdVaJa",
        "target": "section",
        "toString": [Function],
        "warnTooManyClasses": [Function],
        "withComponent": [Function],
      },
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getDerivedStateFromProps": true,
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
  },
}
`;
